File: alice/megamind/example_apps/java/src/main/java/ru/yandex/alice/megamind/cowsay/Application.java
package ru.yandex.alice.megamind.cowsay;

import com.google.protobuf.util.JsonFormat;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;

@SpringBootApplication
public class Application {

    // custom HttpMessageConverter for mm protobuf content-type headers
    @Bean
    CustomProtobufHttpMessageConverter protobufHttpMessageConverter() {
        return new CustomProtobufHttpMessageConverter(JsonFormat.parser(), JsonFormat.printer());
    }

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: alice/divkt_renderer/src/main/java/Application.kt
package ru.yandex.alice.divktrenderer

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Bean
import ru.yandex.alice.paskills.common.logging.protoseq.Setrace

@SpringBootApplication
open class Application {

    @Bean
    open fun setrace() = Setrace("divkt-renderer")

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(Application::class.java, *args)
        }
    }
}
